kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/SubscriberServiceImpl.java

```diff
@@ -33,7 +40,32 @@ public void add(Subscriber subscriber) {
 
     @Override
     public void update(Subscriber subscriber) {
```

Comment:
Yep, org.springframework.transaction.annotation.Transactional. Thanks for the pointer to the POM imports, that fixed the issue. Now, even weirder -- my tests started failing (testUpdateDOB and testUpdateLMP fail because for some reason it looks like the subscriber is getting updated *before* the update call). Curious...